bare-metal/elastic-metal/how-to/use-private-networks.mdx

```diff
@@ -128,16 +128,16 @@ You must configure the virtual network interface on each Elastic Metal server yo
     </Message>
 2. Launch the **Server Manager** application.
 3. Click **Local Server** in the menu on the left. Then click **NIC Teaming** in the **Properties** section.
-2. Launch the **Server Manager** application.am name of your choice (e.g. `default`) and tick the checkbox **NIC1**. Then click **OK**.
+2. Launch the **Server Manager** application.
 4. Click **Tasks** > **New Team**. Enter a team name of your choice (e.g. `default`) and tick the checkbox **NIC1**. Then click **OK**.
-5. Select the **Team interfaces** tab in the **Adapters and Interfaces** section. Click **Tasks** > **Add interface**. A pop-up displays.your VLAN (e.g. 1234). Click **OK** to confirm.
+5. Select the **Team interfaces** tab in the **Adapters and Interfaces** section. Click **Tasks** > **Add interface**. A pop-up displays.
 6. Enter a name for the new interface (e.g. `default - VLAN 1234` if your VLAN ID is 1234). Select **Specific VLAN** and enter the ID of your VLAN (e.g. 1234). Click **OK** to confirm.
     <Message type="important">
       Two network interfaces display: Your VLAN and `default`. Make sure to keep the default network as you will lose access to your server via the public Internet otherwise.
```

Comment:
```suggestion
      Two network interfaces display: Your VLAN and `default`. Make sure to keep the default network, otherwise you will lose access to your server via the public Internet.
```